manual/configuration.md

```diff
@@ -252,13 +260,23 @@ via [#5882](https://github.com/rubocop-hq/rubocop/pull/5882). This change allows
 people to include/exclude precisely what they need to, without the defaults
 getting in the way.
 
-#### Path relativity
+#### Inheritance is configurable
```

Comment:
I think there's already a section about this somewhere in this file. I was thinking just about showing `inherit_mode` to merge `Include` config, as that's a recurring question popping up.